src/routing/authorisedRoute.component.tsx

```diff
@@ -48,7 +48,8 @@ const mapDispatchToProps = (dispatch: Dispatch): WithAuthDispatchProps => ({
 // generator function to create an authentication layer around the given component
 const withAuth =
   (adminSection: boolean) =>
-  <T,>(ComponentToProtect: ComponentType<T>): NamedExoticComponent<T> => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  <T,>(ComponentToProtect: ComponentType<T>): any => {
```

Comment:
@louise-davies this package update resulted in the following error appearing:

![image](https://user-images.githubusercontent.com/71134574/185150309-3f3757d6-ca2c-4aa8-bd4b-7fb10bc18baf.png)

Having hunted through [here](https://stackoverflow.com/questions/53433989/react-memo-with-typescript-in-react), [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/38678) and [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33602#issuecomment-470155095), I still couldn't figure out how to fix this (although I'm sure the answer is in there). Changing the return type to `any` conveniently works but I'm not happy with the solution. Any ideas?